src/StatusQ/Components/StatusChatList.qml

```diff
@@ -96,31 +92,19 @@ Column {
                 StatusChatListItem {
 
                     id: statusChatListItem
-
-                    property string profileImage: ""
-
                     opacity: dragSensor.active ? 0.0 : 1.0
-                    Component.onCompleted: {
-                        if (typeof statusChatList.profileImageFn === "function") {
-                            profileImage = statusChatList.profileImageFn(model.chatId || model.id) || ""
-                        }
-                    }
                     originalOrder: model.position
-                    chatId: model.chatId || model.id
-                    categoryId: model.categoryId || ""
-                    name: !!statusChatList.chatNameFn ? statusChatList.chatNameFn(model) : model.name
-                    type: model.chatType
-                    muted: !!model.muted
-                    hasUnreadMessages: !!model.hasUnreadMessages || model.unviewedMessagesCount > 0
-                    hasMention: model.mentionsCount > 0
-                    badge.value: model.chatType === StatusChatListItem.Type.OneToOneChat ?
-                        model.unviewedMessagesCount || 0 :
-                        model.mentionsCount || 0
-                    selected: (model.chatId || model.id) === statusChatList.selectedChatId
-
-                    icon.color: model.color || ""
-                    image.isIdenticon: !!!profileImage && !!!model.identityImage && !!model.identicon
-                    image.source: profileImage || model.identityImage || model.identicon || ""
+                    chatId: model.itemId
+                    categoryId: model.itemId
```

Comment:
I think this one is wrong. Probably `categoryId`?

Reply:
@jrainville even it looks like it's wrong, actually it's not :) The reason is cause components used on the qml side doesn't follow model's struct from the backend side. And what's the point here... on the backend we have a single model which represents list of chats in case of chat section and list of channels/categories in case of community section. That further means `itemId` in case of chat section will be `chatId`, but in case of community section it will be either `channelId` or `categoryId` (only in case if an item is a category, then it has `subItems` which is a model representing channels belonging to that category). But qml side doesn't follow that struct, old qml components got minimal updates in order to handle new backend model correctly, and qml part recognizes categoryId as a special property, different than chatId/channelId, and that's why the same model item is used on both places.